src/lib/unlang/xlat_builtin.c

```diff
@@ -2744,6 +2750,8 @@ static xlat_action_t xlat_func_pairs(TALLOC_CTX *ctx, fr_dcursor_t *out,
 				&(tmpl_rules_t){
 					.attr = {
 						.dict_def = request->dict,
+						.list_def = fr_request_attr_request,
+						.allow_wildcard = true,
```

Comment:
Hmm, what does allow_wildcard so, and when would we want to disallow it?

Reply:
It allows things such as `&request.[*]` to parse where `allow_unresolved` is false